### ISSUE TITLE
table tbody th top line fix

### ISSUE THREAD

**Issue by dejurin** [mantinedev/mantine#3555]
### What package has an issue

@mantine/core

### Describe the bug

In the last fix, I left out the top line for tag **TH**.
<img width="286" alt="Screenshot 2023-02-15 at 18 12 24" src="https://user-images.githubusercontent.com/4470015/219086407-dabab1f0-f1dd-49d1-8c45-30ca5d53e039.png">

### What version of @mantine/hooks page do you have in package.json?

5.10.4

### Do you know how to fix the issue

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

Change

```SASS
'& tbody tr:first-of-type td': {
    borderTop: 'none',
},
```

To

```SASS
'& tbody tr:first-of-type td, & tbody tr:first-of-type th': {
    borderTop: 'none',
},
```

**Comment by rtivital** [comment]
You are welcome to submit a PR

**Comment by rtivital** [comment]
Fixed in 5.10.5
